Nos/Views/Modifiers/ReportMenuModifier.swift

```diff
@@ -173,16 +196,39 @@ struct ReportMenuModifier: ViewModifier {
         }
     }
     
-    func mute(author: Author) {
-        Task {
-            do {
-                try await author.mute(viewContext: viewContext)
-            } catch {
-                Log.error(error.localizedDescription)
+    func mute(author: Author) async throws {
```

Comment:
Are we doing anything with the error outside of this func? If not, maybe we don't need to throw it. Same question for `unmute` below.
```suggestion
    func mute(author: Author) async {
```